pom.xml

```diff
@@ -174,6 +176,10 @@
                         <exclude>**/*.json</exclude>
                         <exclude>**/*.yml</exclude>
                         <exclude>**/*.yaml</exclude>
+                        <exclude>**/*.sh</exclude>
+                        <exclude>**/Vagrantfile</exclude>
+                        <exclude>**/seaclouds_id_rsa</exclude>
+                        <exclude>**/seaclouds_id_rsa.pub</exclude>
```

Comment:
@andreaturli I found something strange. First `mvn clean install` fails on rat-check on this files, but a second run passes successfully. 

TravisCI also noticed it (line 1221): https://travis-ci.org/SeaCloudsEU/SeaCloudsPlatform/builds/65542052#L1221

Any ideas why this happens?